components/Pagination/sub-components/ActionButton.jsx

```diff
@@ -16,18 +14,18 @@ const ActionButton = styled.a`
   user-select: none;
   white-space: nowrap;
   border-radius: 4px;
-  border: 2px solid transparent;
 
   ${({
     theme: {
       colors: {
         primary: { 100: primary100, 700: primary700 },
       },
-      spacing: { xsmall, xxsmall },
+      spacing: { xxxsmall, xsmall, xxsmall, medium },
     },
   }) => `
+    border: ${xxxsmall}px solid transparent;
     margin: 0 ${xxsmall}px;
-    padding: ${xsmall}px;
+    padding: ${xsmall - xxxsmall}px ${medium}px;
```

Comment:
What do you think in talk with Arthur to align the style guide spacing with the Quantum?
The right choice here is `xxsmal` or `xsmall` so we don't have to do it in that way.

Reply:
Resolved